src/main/java/br/com/alura/comex/ProcessadorDeCsv.java

```diff
@@ -0,0 +1,51 @@
+package br.com.alura.comex;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class ProcessadorDeCsv {
+    public List<Pedido> listarPedidos(String fileName) {
+        ArrayList<Pedido> pedidos = new ArrayList<>();
```

Comment:
Poderia depender da classe concreta apenas na instaciação do objeto!

```suggestion
       List<Pedido> pedidos = new ArrayList<>();
```